classification/src/main/java/dev/ikm/komet/reasoner/ReasonerResultsNode.java

```diff
@@ -113,26 +115,29 @@ public ReasonerResultsNode(ViewProperties viewProperties, KometPreferences nodeP
 				LOG.info("Default ReasonerService: " + this.reasonerService.getName());
 				menuItems.add(new SeparatorMenuItem());
 				{
-					MenuItem item = new MenuItem("Run full reasoner");
-					item.setOnAction(ae -> {
-						runFullReasoner();
+					MenuItem item = new MenuItem("Run reasoner");
+					item.setOnAction(_ -> {
+						runReasoner();
```

Comment:
@emays Out of curiosity could the UI just have a single menu option to Run Reasoner?
The thought was if the system knew it was previously run, couldn't it just run the incremental reasoner?
Does the user really need to know?